### DISCUSSION TITLE
[Spring] Logback을 사용하여 프로젝트 Logging하기

### DISCUSSION THREAD

**Discussion by amaran-th** [BDD-CLUB/challenge-every-week-posting#32] [1. 개발 관련 글 모음]
이번에 서버 배포를 하면서 로깅을 도입해봤는데요,
한 번도 써본 적 없는 라이브러리라서 그런지 자료 조사하는 데에도 시간이 많이 걸렸습니다ㅎㅎ...
조잡한 글이지만 읽어주시면 감사하겠습니당

[블로그 링크](https://amaran-th.github.io/Spring/[Spring]%20Logback%EC%9C%BC%EB%A1%9C%20%EB%A1%9C%EA%B9%85(Logging)%ED%95%98%EA%B8%B0/)

**Comment by shkisme** [comment]
오 로깅~ 저도 `@Slf4j` 은 꽤 써본 기억이 있네요~ 근데 아직 이걸 실용적으로 써본 적이 없어서 궁금했는데 글 잘 읽었습니다!
근데 서버 배포는 어떤 일 때문에 하시나요? 우테코?! 궁금하네용 ㅎㅎ

**Reply by amaran-th** [reply]
전에 키퍼 백엔드 코드 구경하다가 본 것 같네요! 그 땐 보안 관련 어노테이션인줄 알았는데ㅋㅋ

서버배포는 이번에 우테코에서 프론트 팀이랑 협업 미션을 하고 있거든요!
덕분에 aws에서 처음으로 서버를 띄워봤습니다ㅎㅎ
